custom/icds_reports/tests/test_daily_attendence.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+
+from django.test.testcases import TestCase
+from custom.icds_reports.models.aggregate import DailyAttendance, AggregateChildHealthDailyFeedingForms
+
+
+class TestDailyAttendance(TestCase):
```

Comment:
I think these cases are all covered by the regular tests  that compare the CSVs. Adding the data to the fixtures should be enough.